fw/http_parser.c

```diff
@@ -2863,13 +2863,20 @@ __FSM_STATE(st) {							\
 static int
 __parse_etag_or_if_nmatch(TfwHttpMsg *hm, unsigned char *data, size_t len)
 {
-	int weak, r = CSTR_NEQ;
+	int no_quotes, r = CSTR_NEQ;
 	bool if_nmatch = TFW_CONN_TYPE(hm->conn) & Conn_Clnt;
 	__FSM_DECLARE_VARS(hm);
 
+#define ETAG_MOVE_fixup_flag(flag, to)					\
+do {									\
+	__set_bit(flag, msg->flags);					\
+	parser->_i_st = &&to;						\
+	goto to;							\
+} while (0)
```

Comment:
We don't need the macro since it's used in only one place

Reply:
fixed